src/net/throttle_node.h

```diff
@@ -55,8 +55,7 @@ class ThrottleNode {
   typedef std::function<void ()> slot_void;
 
   ThrottleNode(uint32_t rateSpan) : m_rate(rateSpan)  { clear_quota(); }
-  ThrottleNode(const ThrottleNode&) = delete;
-  ThrottleNode& operator=(const ThrottleNode&) = delete;
+  ~ThrottleNode() = default;
```

Comment:
Removing those deleted copy methods is incorrect.

Reply:
added back.